python_modules/dagster/dagster/_core/definitions/time_window_partitions.py

```diff
@@ -1637,10 +1637,13 @@ def serialize(self) -> str:
         return json.dumps(
             {
                 "version": self.SERIALIZATION_VERSION,
-                "time_windows": [
-                    (window.start.timestamp(), window.end.timestamp())
-                    for window in self.included_time_windows
-                ],
+                # sort to ensure that equivalent partition subsets have identical serialized forms
```

Comment:
Are we sure these aren't sorted already?

Reply:
ah yeah they're already sorted -- updated